app/views/ApplicationCompleteView.scala.html

```diff
@@ -110,31 +111,14 @@
 <!--If you have any questions about your application-->
     @h3(messages("applicationComplete.questions.subheading"))
     @paragraph( Html(messages("applicationComplete.questions.para.1",
-advanceValuationRulingServiceEmailLink)
-                ))
+                    advanceValuationRulingServiceEmailLink)
+    ))
 
 
 <!--Appeal a ruling decision-->
     @h3(messages("applicationComplete.appeal.subheading"))
     @paragraph(Html(messages("applicationComplete.appeal.paragraph.1", advanceValuationRulingServiceEmailLink)))
-    @paragraph(Html(messages("applicationComplete.appeal.paragraph.2", appealLink)))
-
-    @subheading(messages("applicationComplete.application.heading"),"l")
-    
-    @if(isIndividual) {
-        @subheading(messages("applicationComplete.applicant.heading"))
-        @govukSummaryList(applicationViewModel.eori)
```

Comment:
I _think_ you can remove `applicationViewModel` from the imports & parameters for this view